src/app/modules/main/chat_section/chat_content/messages/module.nim

```diff
@@ -678,6 +678,7 @@ method resendChatMessage*(self: Module, messageId: string): string =
   return self.controller.resendChatMessage(messageId)
 
 method resetNewMessagesMarker*(self: Module) =
+  self.view.setFirstUnseenMessageLoaded(false)
```

Comment:
I think you should not reset it.
If you'll set it to false, loadingMessagesView will be showing, which is wrong.

Reply:
I don't think that we want to show loading messages view when we are waiting for a setFirstUnseenMessageLoaded signal

Reply:
I am not sure I understand, according to the following logic, it is desired behavior.
https://github.com/status-im/status-desktop/blob/34c04f0af65963c1a3caca880251c9d2c46add29/ui/app/AppLayouts/Chat/views/ChatMessagesView.qml#L166-L167

Could you please elaborate on what use case it breaks?

Reply:
Would be nice to test, when you are scrolling up the chat, and receiving a new message (in order new messages marker appeared). Will you UI flick with loading state or not?

Reply:
This behavior is desired on the first chat opening because the backend needs to load the initial message data. When it is loaded, no cense in this UI, all chats will be opened and rendered quickly